mtp_cashbook/templates/disbursements/start.html

```diff
@@ -11,23 +11,34 @@
   <div class="mtp-disbursement-form-preview grid-row">
     <div class="column-two-thirds">
       <header>
-        <h1 class="heading-xlarge">{% trans 'Digital disbursements' %}</h1>
+        <h1 class="heading-xlarge">{% trans 'Digital disbursements tool' %}</h1>
       </header>
+      <h3><strong>{% trans 'How does it work' %}</strong></h3>
+        <ol class="list list-number">
+          <li>{% blocktrans trimmed %}<strong>The prisoner </strong> fills in the payment request paper form {% endblocktrans %}</li>
+          <li>{% blocktrans trimmed %}<strong>You</strong> enter the payment request details from the form into this tool{% endblocktrans %}</li>
+          <li>{% blocktrans trimmed %}<strong>A business hub colleague</strong> checks the details match{% endblocktrans %}</li>
+          <li>{% blocktrans trimmed %}<strong>SSCL</strong> sends the confirmed payment to the recipient by bank transfer or check{% endblocktrans %}</li>
+        </ol>
 
-      <p>
-        {% trans 'Use this tool to process money requests from prisoners who have used the “Send money by bank transfer or cheque” form.' %}
-      </p>
+        <p><a href="{% url 'submit_ticket' %}">{% trans 'Read more about the process' %}</a> </p>
 
-      <p>
-        {% trans 'Make sure you have the filled form in front of you before you start.' %}
-      </p>
+       <h3><strong>{% trans 'Enter a payment request' %}</strong></h3><br>
 
-      <p>
-        <a href="{{ view.get_success_url }}" class="button button-start" role="button">{% trans 'Start now' %}</a>
-      </p>
+       <p>{% trans 'Use this tool to process a payment request from prisoners.' %}</p>
+
+       <p>{% trans 'Before you start, have the completed form in front of you.' %}</p>
+
+    <div class="grid-row">
```

Comment:
the indentation is a little off here. each open tag should indent it's contents by 2 spaces. it's really just for readability so you can tell what the nesting is more easily